go/ethadapter/geth_rpc_client.go

```diff
@@ -40,6 +40,19 @@ func NewEthClient(config config.HostConfig) (EthClient, error) {
 	}, nil
 }
 
+// NewEthClient instantiates a new ethadapter.EthClient that connects to an ethereum node
+func NewEthClient(ipaddress string, port uint, timeout time.Duration) (EthClient, error) {
```

Comment:
@otherview I don't think it makes sense to have both of these. The idea was that we use a single config object everywhere for setting up all the various parts of the new that need config. But in practice, I can see that an Eth client isn't always set up in the context of a host.

I say that if we're going to add this new constructor, let's just kill off the old one.

Reply:
I'm happy killing the old one, but can we make that a different PR ? 
I think this is big as is.

I'll add a TODO on it if that's ok ?

Reply:
I guess so. IMO it would be cleaner to revert the addition of the new method, use hostconfig everywhere for now, and add your new method in a future PR. Like part of the size of the PR (at least in terms of files touched) is actually due to the addition of this new method and the resulting changes.

But probs a bad move given the urgency. TODO is fine :)